### PR TITLE
feat(docs): docs up to date with master

### LINKED ISSUE #157: write docs

### PR DESCRIPTION
**Pull request by ashleygwilliams**
fixes #157

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,29 +4,72 @@
...
+    Publish your Worker to Cloudflare. This uses several keys in your `wrangler.toml` depending on whether
+    you are publishing to a workers.dev subdomain or your own domain, registered with Cloudflare.
+
+    ```
+    wrangler publish
+    ```
+
+    This command will publish your worker to your workers.dev subdomain. If you have set `private: true` in
+    your `wrangler.toml` it will not make it publicly available. If you have set `private: false` or not
+    set the `private` key, `publish` will make your worker available at `<project-name>.<subdomain>.workers.dev`.
```
[the remark this answers is not in the document]
Comment (xtuc) [reply README.md]: For correctness I would change `private: true` to `private = true`. It also avoid the confusion with the private key in package.json.

File: README.md [hunk 2]
```diff
@@ -4,29 +4,72 @@
 [![crates.io](https://meritbadge.herokuapp.com/wrangler)](https://crates.io/crates/wrangler)
 [![Build Status](https://dev.azure.com/ashleygwilliams/wrangler/_apis/build/status/cloudflare.wrangler?branchName=master)](https://dev.azure.com/ashleygwilliams/wrangler/_build/latest?definitionId=1&branchName=master)
 
-✨ CHECK OUT THE [TUTORIAL](https://developers.cloudflare.com/workers/webassembly/tutorial/) ✨
-
 ## 🎙️ Commands
 
-`wrangler` is a CLI tool designed for folks who are interested in using Rust-generated WebAssembly on
-Cloudflare Workers. This tool gives you the following commands:
+`wrangler` is a CLI tool designed for folks who are interested in using Cloudflare workers.
```
Comment (signalnerve) [inline README.md]: Could we move this into the top-level (above "Commands") and make it slightly more declarative about what `wrangler` is? 

> `wrangler` is a CLI tool for developing and publishing applications with Cloudflare Workers.

Open to copy there, just generally think we could use a one-liner about _what_ wrangler is at the beginning of the README

File: README.md [hunk 3]
```diff
@@ -4,29 +4,72 @@
...
 
   - ### 👯 `generate` 
     Scaffold a project, including boilerplate for a Rust library and a Cloudflare Worker.
     You can pass a name and template to this command optionally.
```
Comment (signalnerve) [inline README.md]:
```suggestion
    You can pass a `name`, `template`, and `type` to this command optionally. 
```

File: README.md [hunk 4]
```diff
@@ -4,29 +4,72 @@
...
+    with the `"type"` declared there.
+
   - ### 🔬 `preview`
     Preview your project using the cloudflareworkers.com API.
+
+    You can optionally pass `get` or `post` and a `body` to this command. This will send a request to your
+    worker on the preview service and return the response in your terminal. For example:
+
+    ```
+    wrangler preview post hello=hello
```
Comment (signalnerve) [inline README.md]: woah! TIL

File: README.md [hunk 5]
```diff
@@ -4,29 +4,72 @@
...
+    wrangler preview get // this is the default
+    ```
+
   - ### ☁️ 🆙 `publish`
-    Publish your Worker and WebAssembly to Cloudflare. This uses the `account_id` and `zone_id`
-    from the `wrangler.toml`.
+
+    Publish your Worker to Cloudflare. This uses several keys in your `wrangler.toml` depending on whether
```
Comment (signalnerve) [inline README.md]:
```suggestion
    Publish your application to Cloudflare Workers. This uses several keys in your `wrangler.toml` depending on whether
```

File: README.md [hunk 6]
```diff
@@ -147,6 +201,14 @@ There are two types of configuration that `wrangler` uses: global user and per p
...
+    To publish to a workers.dev subdomain:
     ```
     wrangler publish
     ```
+
+    To publish to a domain you have registered with Cloudflare, add a `route` and a `zone_id` to your
+    `wrangler.toml`. Then run:
+
+    ```
+    wrangler publish --release
```
Comment (signalnerve) [inline README.md]: OK, neat – so my understanding of this update means that wrangler defaults to a `workers.dev` subdomain deploy, right? To do a zone deploy, you can use `wrangler publish --release`?

**Comment by rita3ko** [comment]
Would be good to add How to update Wrangler to the installation guide (I don't think it's very intuitive / some people might not be familiar with Cargo - my instinct was to try `cargo upgrade`...).

File: README.md (hunk 6 above)
Comment (signalnerve) [reply README.md]: Is it possible to simplify this from a usability standpoint a bit based on what we know from `wrangler.toml`? If a `zone_id` is present, can we add the `--release` flag for the user so they don't have to work out which one to run themselves?

(If I'm totally misunderstanding the entire differentiation between these commands, let me know!)

**Comment by ashleygwilliams** [comment]
leaving install and update . instructions . for #126

**Pull request merged**